src/main/java/de/tum/in/test/api/util/ReflectionTestUtils.java

```diff
@@ -82,6 +83,31 @@ public static Object newInstance(String qualifiedClassName, Object... constructo
 		return newInstance(getClazz(qualifiedClassName), constructorArgs);
 	}
 
+	/**
+	 * Instantiate an object of a class by its qualified name and the constructor
+	 * arguments, if applicable.
+	 * <p>
+	 * This method does not support passing null, passing subclasses of the
+	 * parameter types or invoking constructors with primitive parameters. Use
+	 * {@link #newInstance(Constructor, Object...)} for that.
+	 * <p>
+	 * Forces the access to package-private, {@code protected}, and {@code private}
+	 * constructors. Use {@link #newInstance(String, Object...)} if you do not
+	 * require this functionality.
+	 *
+	 * @param qualifiedClassName The qualified name of the class that needs to get
+	 *                           retrieved (package.classname)
+	 * @param constructorArgs    Parameter instances of the constructor of the
+	 *                           class, that it should use to get instantiated with.
+	 *                           Do not include, if the constructor has no
+	 *                           arguments.
```

Comment:
I would delete this statement here as well. It seems, it was already in the documentation of the codebase and has slipped through my review in the past. Sorry for that. I will remove all of those statements afterwards.